lib/Transport.js

```diff
@@ -5,6 +5,7 @@ var Transport = module.exports = function(path, auth) {
 };
 
 var request = require("superagent");
+var Perry = require("perry");
```

Comment:
Browserify, shims and browser-side URL encoding don't support nested params... so I'm now using Perry. Which works a treat both serverside and browserside.